clang/test/Driver/unsupported-option-gpu.c

```diff
@@ -0,0 +1,7 @@
+/// Some target-specific options are ignored for GPU, so %clang exits with code 0.
+// DEFINE: %{gpu_opts} = --cuda-gpu-arch=sm_60 --cuda-path=%S/Inputs/CUDA/usr/local/cuda --no-cuda-version-check
```

Comment:
`--cuda-gpu-arch` is an alias to `--offload-arch`, probably better to use that one.

Reply:
Thanks for the suggestion. Edited